sdks/python/apache_beam/runners/common.py

```diff
@@ -310,12 +313,14 @@ def _process_outputs(self, element, results):
         windowed_value = WindowedValue(
             result.value, result.timestamp,
             self.window_fn.assign(assign_context))
+        multiplicity = len(windowed_input_element.windows)
       else:
-        windowed_value = element.with_value(result)
-      if tag is None:
-        self.main_receivers.receive(windowed_value)
-      else:
-        self.tagged_receivers[tag].output(windowed_value)
+        windowed_value = windowed_input_element.with_value(result)
+      for _ in range(multiplicity):
+        if tag is None:
+          self.main_receivers.receive(windowed_value)
```

Comment:
Now that we duplicate the results we send to the next Receiver, do we want to have the windowed values we send only have one window?  Otherwise, won't they be duplicated again by any following operations? (See comment in window_test.py).